website/docs/getting-started/template-walkthrough.md

```diff
@@ -108,47 +108,22 @@ new StepFunction(stack, "Workflow", async (event: { name: string }) => {
 There is a lot to unpack in these three lines.
 
 First off, we're able to call the `sayFunction` Construct we just defined as if it were a standard nodejs `function`.
-The types inferred from the closure in the `Function` definition above are preserved giving us type-checking when we call `sayFunction`.
+The types inferred from arrow function in the `Function` definition above are preserved, giving us type-checking when we call `sayFunction`.
 In Functionless, types are maintained across service boundaries, removing a whole class of user error.
 
-"But wait!", you might say. "Don't I have to set up IAM permissions for this Step Function to call that Lambda?"
-
-Fear not.
-Functionless introspects how your runtime code interacts with its infrastructure and uses this information to automatically configure least-privilege policies.
-
-To verify this, you can run `npm run synth` and look at the CloudFormation for the generated workflow policy:
-
-```yaml
-# ...
-WorkflowRoleDefaultPolicy3B788295:
-  Type: AWS::IAM::Policy
-  Properties:
-    PolicyDocument:
-      Statement:
-        - Action: lambda:InvokeFunction
-          Effect: Allow
-          Resource:
-            - Fn::GetAtt:
-                - SayFunction4D0973CB
-                - Arn
-            - Fn::Join:
-                - ""
-                - - Fn::GetAtt:
-                      - SayFunction4D0973CB
-                      - Arn
-                  - :*
-      Version: "2012-10-17"
-    PolicyName: WorkflowRoleDefaultPolicy3B788295
-    Roles:
-      - Ref: WorkflowRole98C7DC98
-  Metadata:
-    aws:cdk:path: MyStack/Workflow/Role/DefaultPolicy/Resource
-# ...
-```
+Secondly, Functionless introspects how your runtime code interacts with its infrastructure and uses this information to automatically configure least-privilege policies.
```

Comment:
> introspects how your runtime code

Should we dumb down the wording a bit?

"Functionless uses your runtime code"
"Functionless creates the policies that your application needs to run, which means your policies with start with [least-privilege](some link) permissions.